roxie/udplib/udpsha.hpp

```diff
@@ -88,27 +88,34 @@ class queue_t
     DataBuffer *head = nullptr;      // We add at tail and remove from head
     DataBuffer *tail = nullptr;
 
-    unsigned count = 0;
+    RelaxedAtomic<unsigned> count{0};       // always updated inside a critical section, by atomic to guarantee it can be read from outside.
```

Comment:
What does "by atomic" mean?

Reply:
@richardkchapman updated the comment and repushed